geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/security/Authorizer.java

```diff
@@ -12,10 +12,10 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.security.internal.server;
+package org.apache.geode.internal.protocol.protobuf.security;
 
 import org.apache.geode.security.ResourcePermission;
 
 public interface Authorizer {
-  boolean authorize(ResourcePermission permissionRequested);
+  boolean authorize(Object userCredBlob, ResourcePermission permissionRequested);
```

Comment:
not sure that `userCredBlob` is a technically correct or acceptable name. Possible provide something more useful like `authenticationSubject` or `authenticationToken` instead?

Reply:
Going with authenticationSubject since that's what we're using in the simple authorizer.

Reply:
Another possibility could be "token" of some sort.